client/interfaces.go

```diff
@@ -89,6 +89,12 @@ type ContainerServer interface {
 	GetContainerMetadata(name string) (*api.ImageMetadata, string, error)
 	SetContainerMetadata(name string, metadata api.ImageMetadata, ETag string) error
 
+	GetContainerTemplateFiles(containerName string) (templates []string, err error)
+	GetContainerTemplateFile(containerName string, templateName string) (content io.ReadCloser, err error)
+	CreateContainerTemplateFile(containerName string, templateName string) (err error)
+	UpdateContainerTemplateFile(containerName string, templateName string, content io.ReadSeeker) (err error)
```

Comment:
Feels to me like Create and Update should both take the same arguments. If someone wants to create an existing file or empty an existing one, they can pass an empty ReadSeeker.